utils/hct/ExtractIRForPassTest.py

```diff
@@ -104,7 +104,7 @@ def main(args):
         # 1. Gets the pass list for an HLSL compilation using -Odump
         cmd = ["dxc", "/Odump", args.hlsl_file] + args.compilation_options
         # print(cmd)
-        all_passes = subprocess.check_output(cmd, text=True)
+        all_passes = subprocess.check_output(cmd, universal_newlines=True)
```

Comment:
Apparently text and universal_newlines are aliases of each other. Any specific reason you opted to switch to universal_newlines?

Reply:
I wasn't sure whether to leave this in and I did fail to mention it in the description. The reason is just that I happened to have an older version of python on a system I was using and the `text` alias was added later, so I thought I'd make it more available